builder/builder.go

```diff
@@ -522,10 +650,39 @@ func (b *Builder) UpdateMixVer() error {
 	return ioutil.WriteFile(filepath.Join(b.Versiondir, "mixversion"), []byte(strconv.Itoa(mixver+10)), 0644)
 }
 
+func (b *Builder) buildMixBundleDir() error {
+	if err := os.MkdirAll(b.Bundledir, 0777); err != nil {
+		return errors.Errorf("Failed to create dir: %s", b.Bundledir)
+	}
+	// TODO: clear it out if it exists
+
+	bundles, err := b.getMixBundlesListFull()
+	if err != nil {
+		return err
+	}
+
+	for _, bundle := range bundles {
+		path, err := b.getBundlePath(bundle)
+		if err != nil {
+			return err
+		}
+		if err = helpers.CopyFile(filepath.Join(b.Bundledir, bundle), path); err != nil {
```

Comment:
I would add a comment here saying you always **want** to overwrite here, which is why you're not using CopyFileNoOverwrite in this case. Is that a correct assumption?

Reply:
I absolutely meant to clear the dir above, so if the code explicitly did that, would you still want the comment? It's a moot point kinda at that point.